study-builder/tenants/register-user-in-pepper.js

```diff
@@ -97,8 +91,18 @@ function (user, context, callback) {
 
         if (context.request.query.invitation_id) {
             pepper_params.invitationId = context.request.query.invitation_id;
+            console.log('Invitation id passed in (via query) = ' + pepper_params.invitationId);
         } else if (context.request.body.invitation_id) {
             pepper_params.invitationId = context.request.body.invitation_id;
+            console.log('Invitation id passed in (via body) = ' + pepper_params.invitationId);
+        }
+
+        if (context.request.query.language) {
```

Comment:
This will pick up the new `language` parameter that frontend is now passing in. CC @fat23cat @charlottedye.